fineract-core/src/main/java/org/apache/fineract/commands/domain/CommandWrapper.java

```diff
@@ -201,7 +201,7 @@ public boolean isNoteResource() {
     }
 
     public boolean isUpdateOfOwnUserDetails(final Long loggedInUserId) {
-        return isUserResource() && isUpdate() && loggedInUserId.equals(this.entityId);
+        return isUserResource() && (isUpdate() || isChangePasswordOperation()) && loggedInUserId.equals(this.entityId);
```

Comment:
`isUpdate()` is not enough here? I wonder changing password is an update operation...